### PR TITLE
Feature/parse lacaixa

### PR DESCRIPTION
**Pull request by Hyunk3l**
@fiunchinho
- Added support for La Caixa
- Now is using Docker native instead of Docker Machine, since native one is available for all platforms

### CODE REVIEW

File: src/Bank/ImportCommand.php [hunk 1]
```diff
@@ -126,4 +133,13 @@ private function printOutput(OutputInterface $output, Activity $activity)
...
         }
     }
+
+    private function createBankParser(InputInterface $input, array $config)
+    {
+        $bank = $input->getArgument('bank');
+        if(empty($config['parsers'][$bank])) {
+            throw new BankParserNotFoundException('Bank parser not found!');
+        }
+        return new $config['parsers'][$bank];
```
Comment (Hyunk3l) [inline src/Bank/ImportCommand.php:143]: Not the best place to do this

File: src/Bank/Parser/Lacaixa.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Bank\Parser;
+
+use Bank\Activity;
+use Bank\Transaction;
+use FilesystemIterator;
+use PHPExcel_IOFactory;
+
+class Lacaixa implements \Bank\Parser
```
Comment (Hyunk3l) [inline src/Bank/Parser/Lacaixa.php:10]: Basically is a copy of the ING one... I'll think a way to avoid duplicate code

**Comment by fiunchinho** [comment]
This is perfect!! Thank you @Hyunk3l. I think we can merge it as it is. What do you think?

**Comment by Hyunk3l** [comment]
Ok 😄👍

**Pull request merged**